### DISCUSSION TITLE
Twinkle fire SOPHOS malware detection in 1.15

### DISCUSSION THREAD

**Discussion by Ciberyan** [xanderfrangos/twinkle-tray#520] [General]
Hello All !
Dont know if I am the only one, Sophos is my coroporate solution for virus
Installation end , seems working but as far as I see, some dll were blocked at the update time
![Twinkle PUA](https://user-images.githubusercontent.com/74042539/206458397-e62f179d-2452-4f12-8c20-9e39a77acc04.jpg)
Fred, FRANCE

**Comment by xanderfrangos** [comment]
Thanks for sharing. 👍

According to the message, 2 files were flagged as a "Potentially Unwanted Application". Those exact files actually are not necessary, as they're leftover files from the ARM64 build of the app. I've made sure that non-ARM builds of v1.15.1 and future versions don't contain those.

**Comment by Ciberyan** [comment]
Thanks for the quick follow up !
That's said sophos is a pain 😱
Fred

Le jeu. 8 déc. 2022, 22:32, Xander Frangos ***@***.***> a
écrit :
